ecmd.go

```diff
@@ -709,15 +705,14 @@ func eq_cmd(t *Text, cp *Cmd) bool {
 
 func nl_cmd(t *Text, cp *Cmd) bool {
 	f := t.file
-	var a Address
 	if cp.addr == nil {
 		// First put it on newline boundaries
-		a = mkaddr(f)
+		a := Address(mkaddr(f))
```

Comment:
It seems to me that some of the type casts, if not most, are unnecessary. What about also running https://github.com/mdempsky/unconvert after running Grind? *Haven't tried that, though, so I don't know what the output might look like.*

Reply:
Done in a second commit. grind apparently had one problematic case that I undid manually. unconvert seems to have also removed some existing unnecessary casts.